.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: python
 dist: trusty
```

Comment:
You will need to upgrade this job to a newer linux distribution

Reply:
Thanks!  That did the trick!